documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -232,11 +232,11 @@ Configure the network's proxy settings.
 
 ===== Boot Order
 
-On the Raspberry Pi4, you can specify whether to boot from USB or network if the SD card isn't inserted. See xref:raspberry-pi.adoc#raspberry-pi-4-bootloader-configuration[this page] for more information.
+On the Raspberry Pi 4, you can specify whether to boot from USB or network if the SD card isn't inserted. See xref:raspberry-pi.adoc#raspberry-pi-4-bootloader-configuration[this page] for more information.
 
 ===== Bootloader Version
 
-On the Raspberry Pi4, you can tell the system to use the very latest boot ROM software, or default to the factory default if the latest version causes problems.
+On the Raspberry Pi 4, you can tell the system to use the very latest boot ROM software, or revert to the factory default if the latest version causes problems.
```

Comment:
This is not correct - there is no way to reset to factory defaults. What you can do is choose the EEPROM image from the default branch, which may or may not be the same as the image that Pi came with originally. Also, you should use the term `bootloader EEPROM`, not `boot ROM`, since that is how the rest of the documentation refers to it. (There is also an actual boot ROM in the SoC).

Reply:
This is merely documenting the options (as-written) in `raspi-config` https://github.com/RPi-Distro/raspi-config/blob/master/raspi-config#L1541
So perhaps you want to submit an upstream PR?

Reply:
> This is merely documenting the options (as-written) in `raspi-config` https://github.com/RPi-Distro/raspi-config/blob/master/raspi-config#L1541 So perhaps you want to submit an upstream PR?

I think if you're going to document the options, it is important to be correct. I suggest this PR be corrected, and I will file a PR for raspi-config.

Reply:
I've filed https://github.com/RPi-Distro/raspi-config/pull/184 to have the wording in raspi-config fixed in this regard, along with some other wording fixes/improvements to how the EEPROM settings are worded.

Reply:
> IMHO it's more important for the documentation to accurately reflect what `raspi-config` is currently saying.

I disagree, but then I'm not in charge ;-)

Consider the case where a user of a Pi 4 had used the 'factory default' option in raspi-config to, they think, reset the bootloader EEPROM back to what it shipped with. They then discover it doesn't do that, so they turn to the documentation, which says that it _does_ do a factory reset. Perplexed, they spend time checking other things, before asking on the forum, only to discover that the 'factory reset' option in raspi-config actually does no such thing. So they have wasted time checking other stuff, where having accurate documentation could have prevented it.

Reply:
Yes, technically there _could_ be a difference between the bootloader EEPROM that a Pi 4 originally shipped with (if the user has an old Pi 4), and the bootloader EEPROM that the 'factory default' option installs. But the 'factory default' option installed by raspi-config / rpi-eeprom-update is the **same** bootloader EEPROM that gets flashed onto _new_ Pi 4s in the factory. So unless you can come up with a concrete example, I suspect most users don't need to know the difference.

Also, AFAIK there's no way of knowing what version of the bootloader EEPROM that a particular Pi 4 _did_ originally ship with, since it might have already been overwritten with a newer version. pinging @timg236 for confirmation.

Reply:
It is more important that the documentation reflects reality.

Reply:
For the second time, please raise a separate issue to discuss this if you feel it is important. This conversation is not relevant to this PR.

Reply:
> For the second time, please raise a separate issue to discuss this if you feel it is important. This conversation is not relevant to this PR.

Apologies - I had not read the edits you made to your reply before I posted - your original message implied you agreed with me, which is obviously not the case.